crates/trigger/src/runtime_config/client_tls.rs

```diff
@@ -22,7 +25,6 @@ pub fn load_certs(
     let mut custom_root_ca_cursor = Cursor::new(contents);
 
     Ok(rustls_pemfile::certs(&mut custom_root_ca_cursor)
-        .into_iter()
         .map(|certs| certs.unwrap())
```

Comment:
I see you didn't write this, but if you change the other `unwrap`s to return errors it would be nice to fix this one too.